dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops

```diff
@@ -85,8 +85,9 @@ kafka.create.outgoing.retry.CWMRX1010W.useraction=Review the error message, the
 # {0} = kafka record data, {1} = exception details
 # 'nack' is the name of a method and should not be translated
 kafka.input.message.nacked.CWMRX1011E=CWMRX1011E: The nack method was called on a message that was created from a Kafka record, indicating that it was not processed successfully. The reactive message stream will be shut down. The record was: {0}. The exception was: {1}.
-kafka.input.message.nacked.CWMRX1011E.explanation=A message from the Kafka connector caused a failure somewhere in the application, resulting in the message being negatively acknowledged. As the message has not been successfully processed, the committed offset has not be advanced for the partition that the message came from. No more messages can be processed from this partition.
-kafka.input.message.nacked.CWMRX1011E.useraction=Review the error message and the messages.log file on the server to identify the problem. To avoid message processing stopping on a failure, implement logic within your application that provides alternative handling of messages that are negatively-acknowledged.
+kafka.input.message.nacked.CWMRX1011E.explanation=A message from the Kafka connector caused a failure somewhere in the application, resulting in the message being negatively acknowledged. As the message was not successfully processed, the committed offset is not advanced for the partition that the message came from. No more messages can be processed from this partition.
+kafka.input.message.nacked.CWMRX1011E.useraction=Review the error message and the messages.log file on the server to identify the problem. To avoid message processing halting on a failure, implement logic in your application that provides alternative handling for messages that are negatively acknowledged.
```

Comment:
```suggestion
kafka.input.message.nacked.CWMRX1011E.useraction=Review the error message and the messages.log file on the server to determine the issue. To prevent the halting of message processing due to a failure, implement logic in your application for alternative handling of messages that are negatively acknowledged.
```